### PR TITLE
14172-Update "How does it work?"

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:*
https://github.com/bcgov/entity/issues/14172

*Description of changes:*
Update views based on UXpin attached. Also a tooltip for "BC based Company" expansion panel.  I had to separate the expansion panels as only one has tooltip.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by ketaki-deodhar** [comment]
@seeker25, Could you please add @severinbeauvais as reviewer?

File: auth-web/package-lock.json
Comment (severinbeauvais) [inline auth-web/package-lock.json:36794]: @seeker25 Should this hash be different?

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (severinbeauvais) [inline auth-web/src/components/auth/common/NumberedCompanyTooltip.vue:8]: No need to change but this could have been a `template` element, which won't get rendered, whereas this `div` is rendered. (Sometimes you need a div for flex but probably not here.)

Don't change unless you're changing something else. This is low priority.

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (severinbeauvais) [inline auth-web/src/components/auth/common/NumberedCompanyTooltip.vue:33]: Local getters can be public since code outside this class can call it safely.

Again, low priority (no impact atm I think). No need to change.

File: auth-web/package-lock.json
Comment (ketaki-deodhar) [reply auth-web/package-lock.json:36794]: Probably this changed with package version update (npm i). Will wait for @seeker25 to reply.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (severinbeauvais) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:36]: You might to change the const arrays to a local getter.

File: auth-web/package-lock.json
Comment (seeker25) [reply auth-web/package-lock.json:36794]: That hash is fine, 
https://github.com/bcgov/sbc-auth/pull/2199

I've updated the tag for FAS just in case.

Comment (severinbeauvais) [reply auth-web/package-lock.json:36794]: Oh, OK, Ketaki wasn't using the latest lock file.

This is fine.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:101]: I'm OK with this code duplication as long as we have a future ticket to remove it -- probably the same ticket to remove the FF after we deploy the new incorporation types to Prod (may need a new ticket under epic 14108).

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:301]: To make it easier to globally search for font sizes, we usually use a CSS variable here.

For this particular size, you'll have to add a new variable here:
https://github.com/bcgov/sbc-auth/blob/main/auth-web/src/assets/scss/theme.scss

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (severinbeauvais) [reply auth-web/src/components/auth/common/NumberedCompanyTooltip.vue:8]: This comment applies to the other Vue files if you do make an update.

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (severinbeauvais) [reply auth-web/src/components/auth/common/NumberedCompanyTooltip.vue:33]: This comment applies to the other Vue files if you do make an update.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:301]: got it. Will do.

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (ketaki-deodhar) [reply auth-web/src/components/auth/common/NumberedCompanyTooltip.vue:33]: Done for all

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:36]: fixed with local getter/computed locally based on condition

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue]: Done

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (ketaki-deodhar) [reply auth-web/src/components/auth/common/NumberedCompanyTooltip.vue]: Done

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:17]: Don't need template element here.

File: auth-web/src/components/auth/common/NumberedCompanyTooltip.vue
Comment (severinbeauvais) [reply auth-web/src/components/auth/common/NumberedCompanyTooltip.vue]: PS You can leave out `public` as it's public by default.

**Review by severinbeauvais — COMMENTED** [review]
A couple of small (optional) comments but otherwise looks good to go.

**Review by severinbeauvais — APPROVED** [review]
I approve these changes, but it would be nice to get an approval from someone on Relationships team.

**Comment by seeker25** [comment]
Looks good to me, as long as you don't mind cleaning it up after the feature flag isn't used anymore. 

@ketaki-deodhar Talk to your product owner, for more permissions on this repo

I've asked Omid directly before, but he said the proper route is to go through product owners

**Review by seeker25 — APPROVED** [review]

**Comment by severinbeauvais** [comment]
@ketaki-deodhar When this merge and deploy finish, verify your changes in Dev. If that's good then let Relationships team know they can deploy this to Test, QA it, and then deploy to Prod according to their release schedule.

**Comment by ketaki-deodhar** [comment]
> Looks good to me, as long as you don't mind cleaning it up after the feature flag isn't used anymore.
> 
> @ketaki-deodhar Talk to your product owner, for more permissions on this repo
> 
> I've asked Omid directly before, but he said the proper route is to go through product owners

@seeker25, yes there is ticket to remove the FF and related code to be done afterwards. About permissions, I will follow up. Thanks

**Comment by ketaki-deodhar** [comment]
> @ketaki-deodhar When this merge and deploy finish, verify your changes in Dev. If that's good then let Relationships team know they can deploy this to Test, QA it, and then deploy to Prod according to their release schedule.

Will do.

**Pull request merged**
